addons/flux-new-service/apps/dev/<service_name>/app-SealedSecret-env.yaml

```diff
@@ -0,0 +1,5 @@
+# This file should be created through devopstools UI
```

Comment:
@sgametrio devops tool repo update and the sealed secret creation are manually managed?